build/pgo-scenarios.yml

```diff
@@ -53,16 +53,14 @@ parameters:
     arguments: --property pgo=default
   - displayName: No PGO
     arguments: --property pgo=disabled --application.environmentVariables COMPlus_JitDisablePGO=1
-  - displayName: OSR
-    arguments: --property pgo=osr --application.environmentVariables DOTNET_TC_QuickJitForLoops=1 --application.environmentVariables DOTNET_TC_OnStackReplacement=1
   - displayName: Dynamic PGO
     arguments: --property pgo=dynamic --application.environmentVariables DOTNET_TieredPGO=1 --application.environmentVariables DOTNET_TC_QuickJitForLoops=1
-  - displayName: Dynamic PGO OSR
-    arguments: --property pgo=dynamic-osr --application.environmentVariables DOTNET_TieredPGO=1 --application.environmentVariables DOTNET_TC_QuickJitForLoops=1 --application.environmentVariables DOTNET_TC_OnStackReplacement=1
   - displayName: Full PGO
     arguments: --property pgo=full --application.environmentVariables DOTNET_TieredPGO=1 --application.environmentVariables DOTNET_TC_QuickJitForLoops=1 --application.environmentVariables DOTNET_ReadyToRun=0
```

Comment:
`DOTNET_TC_QuickJitForLoops=1` is not needed anymore since it's enabled by default now and we don't plan to change it